kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/SidePanels/ResourceSelection.vue

```diff
@@ -712,8 +704,26 @@
         }
       }
 
+      const prevRoute = ref({ name: PageNames.EXAM_CREATION_ROOT });
+
+      // Can only go back w/ the arrow if the last route isn't the root,
+      // otherwise we only allow close
+      const canGoBack = computed(() => {
+        return Boolean(
+          // If we can go back to the section editor (another side panel) we can go back from here
+          // The beforeRouteLeave will catch and ask the user to confirm that navigation if needed
+          (prevRoute.value.name && prevRoute.value.name === PageNames.QUIZ_SECTION_EDITOR) ||
+            // prevRoute.value.name is falsy if we only just loaded, so we'll never show the back
+            // arrow in that case. Here we check if there is a valid prevRoute and if there is,
+            // then we know we can go back if we have a topicId or are showing bookmarks
+            // TODO When we add search it'll probably want to be here too
+            (prevRoute.value.name && (topicId.value || showBookmarks.value)),
```

Comment:
I think its a very edge case, but with this condition, if I reload the page, `prevRoute.value.name` is always empty, and I completely lose the back button and doesnt appear again even if I navigate through the tree:

https://github.com/user-attachments/assets/7d56749e-6cec-4461-a64d-b58390b86c41

Reply:
Thank you for this - it shouldn't show up on page load, but it should once you start navigating for sure